CRM/Upgrade/Incremental/sql/4.7.beta1.mysql.tpl

```diff
@@ -4,4 +4,6 @@ SELECT @parent_id := id from `civicrm_navigation` where name = 'Administration C
 INSERT INTO civicrm_navigation
 ( domain_id, url, label, name, permission, permission_operator, parent_id, is_active, has_separator, weight )
 VALUES
-( {$domainID}, 'civicrm/a/#/status', '{ts escape="sql" skip="true"}System Status{/ts}', 'System Status', 'administer CiviCRM', '', @parent_id, '1', NULL, 0 );
\ No newline at end of file
+( {$domainID}, 'civicrm/a/#/status', '{ts escape="sql" skip="true"}System Status{/ts}', 'System Status', 'administer CiviCRM', '', @parent_id, '1', NULL, 0 );
+
+UPDATE civicrm_contact SET is_deceased = 0 WHERE is_deceased IS NULL;
```

Comment:
note the funny red mark here means that there is no line break afterwards. Ideally it is best to have a line break afterwards - it prevents the funny red character & also helps a bit in patching. I am NOT recommending you fix this as this is a rapidly changing file anyway & it's not worth a separate PR just to do that.

Also, another nice-to-have is adding a comment with the JIRA before any upgrade SQL statements. Once again this is  more a note for the future